### PR TITLE
Nitric Acid changes

### PR DESCRIPTION
**Pull request by Djiq**
## About The Pull Request
Changes the recipe of nitric acid to be fluacid + nitogen + hydrogen peroxide, temperature 480K. 

Buffs nitric acid from toxpower of 2 to 3. I tested it, instakill syringes are still impossible.

## Why It's Good For The Game

Nitric acid is still worse than facid in 99% of situations, this makes sure that an upgrade is actually an upgrade. for gods sake.

## Changelog
:cl:
balance: Nitric acid is slightly better 
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/recipes/toxins.dm [hunk 1]
```diff
@@ -25,8 +25,8 @@
 
 /datum/chemical_reaction/nitracid
 	results = list(/datum/reagent/toxin/acid/nitracid = 2)
-	required_reagents = list(/datum/reagent/toxin/acid/fluacid = 1, /datum/reagent/nitrogen = 1, /datum/reagent/oxygen = 1)
-	required_temp = 380
+	required_reagents = list(/datum/reagent/toxin/acid/fluacid = 1, /datum/reagent/nitrogen = 1, , /datum/reagent/hydrogen_peroxide = 1)
```
Comment (Fikou) [inline code/modules/reagents/chemistry/recipes/toxins.dm:28]:
```suggestion
	required_reagents = list(/datum/reagent/toxin/acid/fluacid = 1, /datum/reagent/nitrogen = 1,  /datum/reagent/hydrogen_peroxide = 1)
```

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 2]
```diff
@@ -872,7 +872,7 @@
 	name = "Nitric acid"
 	description = "Nitric acid is an extremely corrosive chemical substance that violently reacts with living organic tissue."
 	color = "#5050FF"
-	toxpwr = 2
+	toxpwr = 3
 	acidpwr = 5.0
```
Comment (Tlaltecuhtli) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm:876]: why is acidpwr so low? is this intended or missclick?

Comment (EdgeLordExe) [reply code/modules/reagents/chemistry/reagents/toxin_reagents.dm:876]: it was always low

**Comment by tmtmtl30** [comment]
the stuff is already a hassle, don't make it even more of a nightmare to make

**Comment by ATH1909** [comment]
> the stuff is already a hassle, don't make it even more of a nightmare to make

eh, hydrogen peroxide's a 3 basic chem recipe (like mutagen), I think it'll be fine

File: code/modules/reagents/chemistry/recipes/toxins.dm [hunk 3]
```diff
@@ -25,8 +25,8 @@
 
 /datum/chemical_reaction/nitracid
 	results = list(/datum/reagent/toxin/acid/nitracid = 2)
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/recipes/toxins.dm:27]:
```suggestion
	results = list(/datum/reagent/toxin/acid/nitracid = 3)
```

**Comment by EdgeLordExe** [comment]
> the stuff is already a hassle, don't make it even more of a nightmare to make

since when was nitric acid a hassle to make, i can make it in under 30 seconds from roundstart

**Pull request merged**